src/css/base.scss

```diff
@@ -38,13 +29,77 @@ h1 {
   color: #ffebcd;
 }
 
-.body__modal__login-container {
-  padding: 12px 20px;
+.modal__overlay-welcome{
+  color: lightcyan;
+  font-size: 55px;
   display: flex;
-  justify-content: space-around;
-  flex-direction: column;
+  justify-content: center;
+  margin-top: 140px;
+}
+
```

Comment:
You might think about using some mixins here since you have several css attributes listed out for each of these.  That would be an easy refactor opportunity.